Python/ceval.c

```diff
@@ -3060,6 +3060,34 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, PyFrameObject *f, int throwflag)
             goto error;
         }
 
+        case TARGET(MAKE_CELL): {
+            if (GETLOCAL(oparg) != NULL) {
```

Comment:
Handle this in `PyFrame_LocalsToFast` or the tracing code. This should be unconditional (except for errors).

Reply:
Yeah, this is something I'd meant to fix.